app/components/bali/columns/column/component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Bali
+  module Columns
+    module Column
+      class Component < ApplicationViewComponent
+        attr_reader :options
+
+        def initialize(**options)
+          @class = options.delete(:class)
+          @options = options
+        end
+
+        def call
+          tag.div(content, class: "column #{@class}", **options)
+        end
```

Comment:
Aqui el patrón que me gustaría seguir para la definición de la clase por default es el siguiente:

```suggestion
        def initialize(**options)
          @options = prepend_class_name(options, 'column')
        end

        def call
          tag.div(content, **@options)
        end
```

Creo que queda mucho más consizo.